web/src/components/pages/dashboard/challenge/constants.ts

```diff
@@ -43,3 +44,6 @@ export const pilotDates: ChallengeDuration = {
   start: new Date(!isProduction() ? '2019-11-10' : '2019-11-18'),
   end: new Date(!isProduction() ? '2019-11-30' : '2019-12-08'),
 };
+
+export const isEnrolled = (account: UserClient) =>
```

Comment:
This will need to go into `api.ts` as well, though I imagine that'll get cleaned up as we investigate the login bug.